crates/pathfinder/src/rpc/api.rs

```diff
@@ -119,107 +118,108 @@ impl RpcApi {
-
-            // Use internal_server_error to indicate that the process of querying for a particular block failed,
-            // which is not the same as being sure that the block is not in the db.
-            let global_root = match block_hash {
-                BlockHashOrTag::Hash(hash) => GlobalStateTable::get_root_at_block_hash(&tx, hash)
-                    .map_err(internal_server_error)?,
-                BlockHashOrTag::Tag(tag) => match tag {
-                    Tag::Latest => {
-                        GlobalStateTable::get_latest_root(&tx).map_err(internal_server_error)?
-                    }
-                    Tag::Pending => unreachable!("Pending has already been handled above"),
-                },
-            }
-            // Since the db query succeeded in execution, we can now report if the block hash was indeed not found
-            // by using a dedicated error code from the RPC API spec
-            .ok_or_else(|| Error::from(ErrorCode::InvalidBlockHash))?;
-
-            let global_state_tree = GlobalStateTree::load(&tx, global_root)
-                .context("Global state tree")
-                .map_err(internal_server_error)?;
-
-            let contract_state_hash = global_state_tree
-                .get(contract_address)
-                .context("Get contract state hash from global state tree")
-                .map_err(internal_server_error)?;
-
-            // There is a dedicated error code for a non-existent contract in the RPC API spec, so use it.
-            if contract_state_hash.0 == StarkHash::ZERO {
-                return Err(Error::from(ErrorCode::ContractNotFound));
-            }
-
-            let contract_state_root = ContractsStateTable::get_root(&tx, contract_state_hash)
-                .context("Get contract state root")
-                .map_err(internal_server_error)?
-                .ok_or_else(|| {
-                    internal_server_error(anyhow::anyhow!(
-                        "Contract state root not found for contract state hash {}",
-                        contract_state_hash.0
-                    ))
-                })?;
-
-            let contract_state_tree = ContractsStateTree::load(&tx, contract_state_root)
-                .context("Load contract state tree")
-                .map_err(internal_server_error)?;
-
-            // ContractsStateTree::get() will return zero if the value is still not found (and we know the key is valid),
-            // which is consistent with the specification.
-            let storage_val = contract_state_tree
-                .get(key)
-                .context("Get value from contract state tree")
-                .map_err(internal_server_error)?;
-
-            Ok(storage_val)
-        });
-
-        jh.await
-            .context("Database read panic or shutting down")
-            .map_err(internal_server_error)
-            // flatten is unstable
-            .and_then(|x| x)
+        todo!("Fix to use new schema tables");
```

Comment:
Maybe I should set these to `#[ignore]` so that CI passes? Although I kind of like the more active reminder.

Reply:
Or commented out, doesn't matter.